structured_prediction_baselines/modules/loss/inference_net_loss.py

```diff
@@ -168,9 +168,7 @@ def _get_values(
         )  # if you call this loss, labels cannot be None
 
         if y_cost_aug is None:
-            y_cost_aug = (
-                y_hat if not self.normalize_y else self.normalize(y_hat)
-            )
+            y_cost_aug = torch.zeros_like(y_hat)
```

Comment:
This will break things. Any reason for making y_cost_aug zero instead of assigning it the same value as y_hat?

Reply:
@leejayyoon and I discussed about this, I think it was specific for the case when there was no cost_augmented_layer and we needed inference_net loss from only y_hat.

Can you specify where exactly will it break things?

Reply:
Yes. That is the case I am talking about. Now we are not experimenting using that case but why change it to zero.

Reply:
I guess it is fine. We can change it.